dialog/fileitem_test.go

```diff
@@ -15,34 +16,34 @@ func TestFileItem_Name(t *testing.T) {
 	f := &fileDialog{file: &FileDialog{}}
 	_ = f.makeUI()
 
-	item := f.newFileItem("/path/to/filename.txt", false)
+	item := f.newFileItem(storage.NewURI("file:///path/to/filename.txt"), false)
```

Comment:
Did we agree to storage.NewFileURI()? this make it look pretty darn useful :)

Reply:
We didn't agree to that in particular, but I would be happy to add it. I agree it would come in quite handy.

Reply:
Ah yes I remember now there was potential confusion because we use the word File in other URI places that might need tidying up...
Your call

Reply:
I agree that we should do this, but I think that is out of scope for this PR.